### PR TITLE
Use fastlane for diawi, slack & github comments 🌤

### PR DESCRIPTION
**Pull request by mandrigin**
in effort to minimize `curl` usages in Jenkinsfiles

### 1. upload releases (iOS and Android) to diawi using fastlane
use plugin [`diawi`](https://rubygems.org/gems/fastlane-plugin-diawi) to do this magic

### 2. comment to slack using fastlane
comment to Slack through fastlane [`slack`](https://docs.fastlane.tools/actions/slack/) action

### 3. comment to github using fastlane
replace curl usages with [`github_api`](https://docs.fastlane.tools/actions/github_api/) to comment on GitHub

status: ready

### CODE REVIEW

**Review by jakubgs — APPROVED** [review]
Amazing work, thank you.

File: fastlane/Fastfile [hunk 1]
```diff
@@ -122,6 +163,19 @@ platform :ios do
   lane :clean do
     clean_testflight_testers
   end
+
+  desc "`fastlane ios upload-diawi` - upload .ipa to diawi"
+  desc "also notifies in a GitHub comments and in Slack #jenkins channel"
+  desc "expects to have an .ipa prepared: `status-adhoc/StatusIm.ipa`"
+  desc "expects to have diawi token as DIAWI_TOKEN  env variable"
+  desc "expects to have diawi token as GITHUB_TOKEN env variable"
```
Comment (pedropombeiro) [inline fastlane/Fastfile]: `expects to have diawi token` is repeated 3 times

File: fastlane/Fastfile [hunk 2]
```diff
@@ -153,4 +207,17 @@ platform :android do
         slack_url: ENV["SLACK_URL"]
     )
   end
+
+  desc "`fastlane android upload-diawi` - upload .ipa to diawi"
+  desc "also notifies in a GitHub comments and in Slack #jenkins channel"
+  desc "expects to have an .apk prepared: `android/app/build/outputs/apk/release/app-release.apk`"
+  desc "expects to have diawi token as DIAWI_TOKEN  env variable"
+  desc "expects to have diawi token as GITHUB_TOKEN env variable"
```
Comment (pedropombeiro) [inline fastlane/Fastfile]: Same here

**Review by pedropombeiro — APPROVED** [review]
Looks great, just a minor comment

File: fastlane/Fastfile (hunk 1 above)
Comment (mandrigin) [reply fastlane/Fastfile]: ah, ok
I will fix it and merge after that then :)

File: fastlane/Fastfile (hunk 2 above)
Comment (mandrigin) [reply fastlane/Fastfile]: 👍

**Review by flexsurfer — APPROVED** [review]

**Comment by mandrigin** [comment]
skipping autotests because no code changes are there

**Pull request merged**